### ISSUE TITLE
I would like to contribute and seek for additional contributors.

### ISSUE THREAD

**Issue by adriatic** [gothinkster/aurelia-realworld-example-app#8]
Hello @loaded02 and @EricSimons 

While I have met Eric in the context of discussing the aurelia version of his other key project), I "met" Moritz in the recent Rob's **[blog](http://aurelia.io/blog/2018/04/04/aurelia-realworld-app)**. 

### Executive summary 
Conduit is the ideal application to be used in our own project - writing application development guides for Aurelia community. At the same tome, the side-effect of this documentation work could benefit Conduit as well.

### Long story 
For some time I am trying to "rally" Aurelia community around the idea to create the "additional" type of technical documentation, which "builds on top" of the existing http://aurelia.io/docs components. By this I am referencing the a set of Guides, written by Aurelia user's community members, addressing the "higher level" of application development with Aurelia, described **[here](https://discourse.aurelia.io/t/what-in-the-world-is-the-mission-of-aucs/652)**

My initial approach to create such guides was based on my incorrect assumption that all folks that initially responded to my **[pitch](https://discourse.aurelia.io/t/get-aurelia-great-gag-call-for-action/574/41)** will actually do the work. So, I spent a lot of time building the "tooling" needed to organize a team of 25 developers; when this was finished, the project was rejected by all members (too complicated, I do not have the time to learn this tooling, I do not have the time at the moment, etc)

**[Longarone](https://github.com/longarone)** is my second attempt at this, currently focused on building the set of **[code samples](https://github.com/longarone/code-samples)**, designed to be used as the "back-bone" for the planned Guides which will be using these samples to illustrate many different concepts: app deployment, CI-CD, Identity Management (aka authentication and authorization), etc. These samples start with Aurelia CLI built aurelia **[skeleton samples](https://github.com/longarone/code-samples/tree/master/basic)** , being used in the introductionary chapters. 

The next step is envisioned by presenting all that information using a *real world apps" as a model - as you may realize, I started this idea by using **[Aureliacasts blog](https://github.com/longarone/code-samples/tree/master/basic/basic-blog)** as one of the apps of such more complex types. It ought to be pretty clear why am I writing this text, by now 😄 as Conduit is a perfect candidate for such model, where I can focus on some specific aspects (Identity Management, Serverless, and other backends, etc.

I am also very interested in adding a few more "real world applications" to your original and my Longarone projects (Chat / Video Chat, would be a good example).

**Comment by loaded02** [comment]
@adriatic nice to meet you too. As far as I understand is that you want to contribute to this project and I / we can contribute to yours. That sounds good. I want to create a tutorial for this project similar to [this](https://thinkster.io/angularjs-es6-tutorial). We cant enhance this project with things like video chat, cause the spec doesnt say so. This would have to be another project. Whoever is interested please bring in your ideas for the tutorial.

**Comment by adriatic** [comment]
Hello @loaded02 (Moritz) 

Thanks for such quick and positive response. I have a strong preference for partnering with you (perceived as a "proxy" for **[aurelia-realworld-example-app](https://github.com/gothinkster/aurelia-realworld-example-app)**, because I care about using this blog not only as a "subject" for the Longarone's real world app guides, but also as a tool to be embedded in Longarone's portal (shown below)

![image](https://user-images.githubusercontent.com/2712405/39070328-ff89d24c-44b0-11e8-87b8-e8300accc445.png)

This portal uses Ghost at the moment, and as my first action today, I will replace it with Conduit, so that this Aurelia User Community Space portal will be using our own dog food. 😏 . I will also write the guides for two different back ends - **[serverless](https://serverless.com/)** being one of them.

Since I am very keen on publishing a lot on the authentication / authorization subject, I also want to add the support for several Identity Management PaaS - as options for Conduit.

### To summarize:

- All features that I plan to add will be made available to you in the form of various PRs (hopefully we could also "push" these features further "upstream" so that they do not exist only for Aurelia implementation of Conduit.

- The chat app I mentioned will be built the same way as Conduit - on top of a platform independent specification, where the first implementation will be Aurelia Front End and PubNub back end.

Can we agree on such "draft of a plan"?

**Comment by loaded02** [comment]
Thanks! We could enhance the conduit in various branches if you like. But my goal would be a tutorial.

**Comment by adriatic** [comment]
My comments **[above](https://github.com/gothinkster/aurelia-realworld-example-app/issues/8#issuecomment-383204085)** were made only as "thoughts sharing", as I have no intent to make you do anything outside of your plans, Moritz. Any and all enhancements will be made in a branch that I will use to host Conduit for Aurelia community blogs
